spring-cloud-dataflow-admin-local/src/test/java/org/springframework/cloud/dataflow/admin/spi/local/AdminDefaultTestEnvironmentPostProcessor.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.admin.spi.local;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.env.ConfigurableEnvironment;
+
+/**
+ * @author Ilayaperumal Gopinathan
+ */
+public class AdminDefaultTestEnvironmentPostProcessor implements EnvironmentPostProcessor {
```

Comment:
The problem here is that the `defaultProperties` property source is available for the `module` being deployed because the module context also gets `AdminDefaultEnvironmentPostProcessor` from `s-c-d-admin-starter` **only** in test as the test classpath also includes `s-c-d-admin-starter`. The `defaultProperties` property source shouldn't be available at first place for the modules inside the test.

There is another option to remove the `s-c-d-admin-starter` via maven sure-fire plugin `excludeClasspathComponent` but we would have to skip the `LocalConfigurationTests`.